docs/pages/more/expo-cli.mdx

```diff
@@ -503,6 +503,7 @@ From here, you can choose to generate basic project files like:
 | `EXPO_USE_METRO_WORKSPACE_ROOT`     | **boolean** | Enable auto server root detection for Metro. This will change the server root to the workspace root. Useful for monorepos.                                                                                                                                                                                                                                                           |
 | `EXPO_USE_UNSTABLE_DEBUGGER`        | **boolean** | <StatusTag status="experimental" /><br/>Enable the experimental debugger from React Native.                                                                                                                                                                                                                                                                                          |
 | `EXPO_ADB_USER`                     | **string**  | <StatusTag status="SDK 50+" /><br/>Set the `user` number that should be passed to `--user` with ADB commands. Used for installing APKs on Android devices with multiple profiles. Defaults to `0`.                                                                                                                                                                                   |
+| `EXPO_NO_TELEMETRY_DETACH`          | **boolean** | <StatusTag status="SDK 51+" /><br/>Send telemetry events in the main thread of `@expo/cli`. This slows the CLI down, as it has to wait until all events are sent.                                                                                                                                                                                                                    |
```

Comment:
The same info is being added in this PR: https://github.com/expo/expo/pull/27792/files.

Reply:
It's because I rebased the base PR on `main` (updates workflow was failing). I need to rebase every individual PR to get rid of it, but TL;DR; it's only added in the main PR #27787